schema/me.js

```diff
@@ -0,0 +1,32 @@
+import date from './fields/date';
+import gravity from '../lib/loaders/gravity';
+import Profile from './profile';
+import {
+  GraphQLString,
+  GraphQLObjectType,
+} from 'graphql';
+
+const Me = new GraphQLObjectType({
+  name: 'Me',
+  fields: {
+    id: {
+      type: GraphQLString,
+    },
+    created_at: date,
+    email: {
+      type: GraphQLString,
+    },
+    profile: {
+      type: Profile.type,
+      resolve: ({ default_profile_id }) =>
+        gravity(`profile/${default_profile_id}`),
+    },
+  },
+});
+
+export default {
+  type: Me,
+  resolve: (root, options, { rootValue: { accessToken } }) => {
```

Comment:
So yah—we can pass whatever we want down in the `rootValue` on a per-request basis, and that gets included as the third arg on every `resolve`. So we just pull out the `accessToken` and authenticate the gravity call with it. I think we could pass a single preauthenticated loader down too; but that can be a later refactoring; I was having a little difficulty figuring out how to safely clear it, so this seemed like the simplest first step